### PR TITLE
timelapze 1.0.0 (new cask)

### PR DESCRIPTION
**Pull request by wkaisertexas**
Adds wkaisertexas/screentimelapse to homebrew

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/t/timelapze.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+cask "timelapze" do
+  version "1.0.0"
+  sha256 :no_check
```
Comment (razvanazamfirei) [inline Casks/t/timelapze.rb:3]: Since the download link is versioned, this should have a value.

File: Casks/t/timelapze.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+  url "https://github.com/wkaisertexas/ScreenTimeLapse/releases/download/v1.0.0/TimeLapze.zip"
```
Comment (razvanazamfirei) [inline Casks/t/timelapze.rb:5]:
```suggestion
  url "https://github.com/wkaisertexas/ScreenTimeLapse/releases/download/v#{version}/TimeLapze.zip"
```

File: Casks/t/timelapze.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  name "TimeLapze"
+  desc "Record screen time lapses with ease in a simple, intuitive interface"
```
Comment (razvanazamfirei) [inline Casks/t/timelapze.rb:7]: Make sure the description matches our [guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

**Review by razvanazamfirei — DISMISSED** [review]

**Comment by miccal** [comment]
Thank you @wkaisertexas.

**Review by miccal — APPROVED** [review]

**Comment by wkaisertexas** [comment]
@miccal Thank you!

**Comment by miccal** [comment]
> @miccal Thank you!

Thanks to @razvanazamfirei also for the review.

**Pull request merged**